app/components/permitted-stacks-by-role/component.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  permittedStacks: function() {
+    return this.get('stacks').filter((stack) => {
+      if(stack.get('data.links.organization') !== this.get('role.data.links.organization')) {
+        return false;
+      }
+
+      if(this.get('role.privileged')) {
+        return true;
+      }
+
+      return stack.get('permissions').any((permission) => {
+        if(permission.get('data.links.role') === this.get('role.data.links.self')) {
+          return [this.get('scope'), 'manage'].indexOf(permission.get('scope')) > -1;
+        }
+      });
+    }).map(s => s.get('handle')).join(', ');
+  }.property('scope', 'stacks', 'role.privileged')
```

Comment:
this should probably depend on `stacks.[]` instead of `stacks`. The former observes membership, and the latter only whether the array/object is the same reference.